tests/unit/AnnualReport.spec.ts

```diff
@@ -888,20 +904,24 @@ describe('Annual Report - Part 3 - Submitting', () => {
     const vm = wrapper.vm as any
 
     // make sure form is validated
-    vm.agmDateValid = true
-    vm.addressesFormValid = true
-    vm.directorFormValid = true
-    vm.certifyFormValid = true
-    vm.directorEditInProgress = false
+    await wrapper.setData({
+      agmDateValid: true,
+      addressesFormValid: true,
+      directorFormValid: true,
+      certifyFormValid: true,
+      directorEditInProgress: false
+    })
     store.state.filingData = [{ filingTypeCode: 'OTCDR', entityType: 'CP' }] // dummy data
```

Comment:
I didn't change all of these to `$store.commit()` because some of them work fine as-is.